pipelines/h6c/idr2/v3/lstbin/lstbin-template.toml

```diff
@@ -38,7 +38,7 @@ timeout = "48h"
 outdir = "${LSTAVG_OPTS:outdir}"
 bl_chunk_size = 2000
 parallelize = true
-
+isvalidation = "0"
```

Comment:
Is there a reason this is `"0"` and not `false`

Reply:
Because this parameter gets passed to the do-script, rather than the notebook itself (it just controls where the output HTML/ipynb will be written to). I could make it false and then I think it would be passed as a string to the bash script as "false", but I thought it more clear to test it as 0 vs 1.

Reply:
My preference would be for the do_script work a little harder so that the toml is easier to parse and edit